### ISSUE TITLE
Can't select a buffer

### ISSUE THREAD

**Issue by therobotguyllc** [chilipeppr/workspace-tinyg#6]
I have a Due/tinyg2 board that I setup at work using chilipeppr but it would not work on my home PC.  The specific problem that I have is that the two drop-down menus for the USB connect don't work well. 

The Baud selection lets me set it if I start the server but leave the USB cable unplugged. However, I cannot select the tinygg2 buffer no matter what I do. I've tried multiple browsers. My port server seems to be working just fine. I just cant tell Chilipeppr what tinyg version I am using. It's permanently stuck on "default".

Any suggestions would be helpful. 

![image](https://user-images.githubusercontent.com/19788670/70487073-1d8d5280-1ac2-11ea-8d85-86371f171265.png)

**Comment by chilipeppr** [comment]
Typically when it's a due you get a nice icon next to the com port. Your
screenshot does not show a nice icon. If the USB cable is unplugged,
nothing should show up in that list. So something doesn't sound right. That
buffer pulldown menu I've never heard not work, although that menu does get
locked once you're connected, but you aren't connected to the com port in
that screenshot. Are you sure your due is com 14?

**Comment by therobotguyllc** [comment]
I discovered that the two comm ports on the Due are not the same. It turns out that I was using the wrong port. Once I used the correct port I was able to use the board and my PC recognized it just fine. While plugged into the wrong port I was able to see it connected as a Due board. When connected to the other port it identified as a tinyg2. 

I think it shows that this is my first experiment with the Due .....  -_-
Sorry for the bother.
